posh-awsp.psm1

```diff
@@ -287,8 +392,9 @@ function Read-MenuSelection {
   #
   for($i = 0; $i -lt $Items.Length; $i++) {
     $Indicator = if ($CurrentIndex -eq $i) { "*" } else { " " }
+    $Name = if ($CurrentIndex -eq $i) { "`e[36m$($Items[$i])`e[0m" } else { "$($Items[$i])" }
```

Comment:
Using ANSI colors to highlight the selected profile. https://www.reddit.com/r/PowerShell/comments/b06gtw/til_that_powershell_can_do_colors/

Not required change and can revert. Looks cool tho.